ethereum/contracts/ETHApp.sol

```diff
@@ -83,7 +83,8 @@ contract ETHApp is RewardSource, AccessControl {
         );
 
         balance = balance - _amount;
-        _recipient.transfer(_amount);
+        (bool success, ) = _recipient.call{value: _amount}("");
+        require(success, "Unable to send Ether");
```

Comment:
I think we need to just double check on this line:
 - if the send fails, and the message is within a batch of 10 other messages, will this message break the whole batch or not?

I think not and this is likely fine - I do recall creating a test for this scenario too.

Can you double check @sumitsnk ?